### PR TITLE
Add missing test for GetNodeHostIP

### PR DESCRIPTION
**Pull request by dineshba**
**What type of PR is this?**
/kind cleanup
**What this PR does / why we need it**:
Increases test coverage

### CODE REVIEW

File: pkg/util/node/node_test.go [hunk 1]
```diff
@@ -120,3 +121,61 @@ func TestGetHostname(t *testing.T) {
...
+			description: "with external and internal address",
+			addresses: []v1.NodeAddress{
+				{Type: v1.NodeExternalIP, Address: "1.2.3.4"},
+				{Type: v1.NodeInternalIP, Address: "5.6.7.8"},
+			},
+			expectedHostIP: net.ParseIP("5.6.7.8"),
+			expectError:    false,
+		},
+		{
+			description: "with mutliple external and internal addresses",
```
Comment (johscheuer) [inline pkg/util/node/node_test.go]:
```suggestion
			description: "with external and multiple internal addresses",
```

Comment (johscheuer) [reply pkg/util/node/node_test.go]: nitpicK: `mutliple `

File: pkg/util/node/node_test.go [hunk 2]
```diff
@@ -120,3 +121,61 @@ func TestGetHostname(t *testing.T) {
...
+			addresses: []v1.NodeAddress{
+				{Type: v1.NodeExternalIP, Address: "1.2.3.4"},
+				{Type: v1.NodeInternalIP, Address: "5.6.7.8"},
+				{Type: v1.NodeInternalIP, Address: "3.4.5.6"},
+			},
+			expectedHostIP: net.ParseIP("5.6.7.8"),
+			expectError:    false,
+		},
```
Comment (johscheuer) [inline pkg/util/node/node_test.go]: Could you add some IPv6 test cases ?

File: pkg/util/node/node_test.go (hunk 1 above)
Comment (dineshba) [reply pkg/util/node/node_test.go]: Fixed. Thanks @johscheuer

File: pkg/util/node/node_test.go (hunk 2 above)
Comment (dineshba) [reply pkg/util/node/node_test.go]: Added test with IPv6

**Comment by johscheuer** [comment]
I think I won't be able to approve this pr

**Pull request closed without merging**